### PR TITLE
IH-186 Update sending Alimtalk messages for USER_REGISTER_MEMBERSHIP_PACKAGE

### PR DESCRIPTION
**Pull request by Sotatek-KimMai**

### CODE REVIEW

File: src/main/java/art/heredium/service/MembershipPaymentService.java [hunk 1]
```diff
@@ -132,9 +131,15 @@ private String buildCouponDetails(List<CouponUsage> coupons) {
...
-                    coupon.getCoupon().getDiscountPercent(),
+                    "할인혜택",
+                    coupon.getCoupon().getCouponType().getDesc(),
+                    coupon.getCoupon().getDiscountPercent() != 100
+                        ? coupon.getCoupon().getDiscountPercent() + "%"
+                        : "무료",
+                    "사용횟수",
                     Boolean.TRUE.equals(coupon.getCoupon().getIsPermanent())
                         ? "상시할인"
                         : coupon.getCoupon().getNumberOfUses()))
```
Comment (Sotatek-ThoLe) [inline src/main/java/art/heredium/service/MembershipPaymentService.java:145]:
```suggestion
                        : coupon.getCoupon().getNumberOfUses() + "회"))
```

File: src/main/java/art/heredium/controller/test/AlimTalkTestService.java [hunk 2]
```diff
@@ -153,6 +153,21 @@ private void sendMockedMembershipRegistrationMessageToAlimTalk(String phone) {
...
+        "무료",
+        "사용횟수",
+        "상시할인",
+        "쿠폰명",
+        "coupon 2",
+        "할인혜택",
+        CouponType.ARTSHOP.getDesc(),
+        "50%",
+        "사용횟수",
+        "3");
```
Comment (Sotatek-ThoLe) [inline src/main/java/art/heredium/controller/test/AlimTalkTestService.java:171]:
```suggestion
        "3회");
```

Comment (Sotatek-KimMai) [reply src/main/java/art/heredium/controller/test/AlimTalkTestService.java]: Tôi đã fix: d1f0a379a805c54ae687393815455ecace53561e

File: src/main/java/art/heredium/service/MembershipPaymentService.java (hunk 1 above)
Comment (Sotatek-KimMai) [reply src/main/java/art/heredium/service/MembershipPaymentService.java]: Tôi đã fix: d1f0a379a805c54ae687393815455ecace53561e

**Review by Sotatek-ThoLe — APPROVED** [review]

**Pull request merged**